test/soap/DataService_model_array.wsdl

```diff
@@ -0,0 +1,61 @@
+<wsdl:definitions xmlns:apachesoap="http://xml.apache.org/xml-soap" xmlns:impl="http://stuff.dom" xmlns:intf="http://stuff.dom" xmlns:wsdl="http://schemas.xmlsoap.org/wsdl/" xmlns:wsdlsoap="http://schemas.xmlsoap.org/wsdl/soap/" xmlns:xsd="http://www.w3.org/2001/XMLSchema" targetNamespace="http://stuff.dom">
```

Comment:
nitpick: is it possible to format the wsdl?

Reply:
@dhmlau 

> nitpick: is it possible to format the wsdl? 


done  :)